client/src/listings/Listings.js

```diff
@@ -41,28 +48,40 @@ function Listings(props) {
   const [subcategoryId, setSubcategoryId] = useState('');
   const [categoryOptions, setCategoryOptions] = useState([]);
   const [subcategoryOptions, setSubcategoryOptions] = useState({});
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(15);
```

Comment:
This should not be a state variable, it should just be a constant. Its value never changes.